config/initializers/devise.rb

```diff
@@ -184,7 +184,7 @@
   # Email regex used to validate email formats. It simply asserts that
   # one (and only one) @ exists in the given string. This is mainly
   # to give user feedback and not to assert the e-mail validity.
-  config.email_regexp = /[a-z0-9._%+-]+@[a-z0-9-]+\.[a-z]{2,63}/
+  config.email_regexp = /[a-z0-9._%+-]+@[a-z0-9-]+(\.[a-z]+)*\.[a-z]{2,63}/
```

Comment:
I wonder how this might compare to say:

`/[a-z0-9._%+-]+@[a-z0-9-]+\.([a-z]{2,63})+/` which re-uses one of the existing segments

Reply:
Works as well, updated. Thanks.